static/styles/night_mode.css

```diff
@@ -10,6 +10,27 @@ body.night-mode {
         color: inherit;
     }
 
+    .progressive-table-wrapper {
```

Comment:
Do we really need all this css in this file?

Reply:
night_mode.css just overrides all the other css files.

Reply:
No, I have removed all changes that were overriding.